src/raytracer/xmlreader.m

```diff
@@ -1,55 +1,55 @@
+% entire notice. You may improve,  modify and create derivative works of the software or
+% any portion of the software,  and you may copy and distribute such modifications or
 % works. Modified works should carry a notice stating that you changed the software
 % and should note the date and nature of any such change. Please explicitly
 % acknowledge the National Institute of Standards and Technology as the source of the
 % software.
 %
 % NIST-developed software is expressly provided "AS IS." NIST MAKES NO
-% WARRANTY OF ANY KIND, EXPRESS, IMPLIED, IN FACT OR ARISING BY
-% OPERATION OF LAW, INCLUDING, WITHOUT LIMITATION, THE IMPLIED
-% WARRANTY OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE,
+% WARRANTY OF ANY KIND,  EXPRESS,  IMPLIED,  IN FACT OR ARISING BY
+% OPERATION OF LAW,  INCLUDING,  WITHOUT LIMITATION,  THE IMPLIED
+% WARRANTY OF MERCHANTABILITY,  FITNESS FOR A PARTICULAR PURPOSE, 
 % NON-INFRINGEMENT AND DATA ACCURACY. NIST NEITHER REPRESENTS
 % NOR WARRANTS THAT THE OPERATION OF THE SOFTWARE WILL BE
-% UNINTERRUPTED OR ERROR-FREE, OR THAT ANY DEFECTS WILL BE
+% UNINTERRUPTED OR ERROR-FREE,  OR THAT ANY DEFECTS WILL BE
 % CORRECTED. NIST DOES NOT WARRANT OR MAKE ANY REPRESENTATIONS
-% REGARDING THE USE OF THE SOFTWARE OR THE RESULTS THEREOF,
-% INCLUDING BUT NOT LIMITED TO THE CORRECTNESS, ACCURACY,
-% RELIABILITY, OR USEFULNESS OF THE SOFTWARE.
+% REGARDING THE USE OF THE SOFTWARE OR THE RESULTS THEREOF, 
+% INCLUDING BUT NOT LIMITED TO THE CORRECTNESS,  ACCURACY, 
+% RELIABILITY,  OR USEFULNESS OF THE SOFTWARE.
 %
 % You are solely responsible for determining the appropriateness of using
-% anddistributing the software and you assume all risks associated with its use, including
-% but not limited to the risks and costs of program errors, compliance with applicable
-% laws, damage to or loss of data, programs or equipment, and the unavailability or
+% anddistributing the software and you assume all risks associated with its use,  including
+% but not limited to the risks and costs of program errors,  compliance with applicable
+% laws,  damage to or loss of data,  programs or equipment,  and the unavailability or
 % interruption of operation. This software is not intended to be used in any situation
 % where a failure could cause risk of injury or damage to property. The software
 % developed by NIST employees is not subject to copyright protection within the United
 % States.
 %
-% Modified by: Mattia Lecci <leccimat@dei.unipd.it>, Used MATLAB functions instead of custom ones,
-%    improved MaterialLibrary access, readibility, performance in general
-% Modified by: Neeraj Varshney <neeraj.varshney@nist.gov>, support multiple
+% Modified by: Mattia Lecci <leccimat@dei.unipd.it>,  Used MATLAB functions instead of custom ones, 
+%    improved MaterialLibrary access,  readibility,  performance in general
+% Modified by: Neeraj Varshney <neeraj.varshney@nist.gov>,  support multiple
 % objects and different length units in amf file
```

Comment:
why did you add double spaces? please remove them

Reply:
Opps I replaced "," by ",[space]". My bad. Does it make any difference if we add or remove space between commas? but No problem I will remove it now. :)

Reply:
having a space after a comma does not affect the code, but it improves its readability and it follows a typical coding style.
This, of course, only when you don't do it in comments as well :)

Reply:
:)